src/test/java/org/jilt/test/data/annotations/NonParameterAnnotation.java

```diff
@@ -0,0 +1,8 @@
+package org.jilt.test.data.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Target;
+
+@Target({ElementType.FIELD})
```

Comment:
I think you can drop the `{}` with one element?

```suggestion
@Target(ElementType.FIELD)
```